client/src/components/admin/Sidebar/index.tsx

```diff
@@ -16,8 +16,8 @@ const Index = () => {
   const pathname = usePathname();
 
   const menuItems: MenuItem[] = [
-    {label: '대시보드', href: `${URLS.admin}${URLS.dashboard}`, icon: '📊'},
-    {label: '문서 관리', href: `${URLS.admin}${URLS.documents}`, icon: '📄'},
+    {label: '대시보드', href: `${URLS.admin}${URLS.dashboard}` as Route, icon: '📊'},
+    {label: '문서 관리', href: `${URLS.admin}${URLS.documents}` as Route, icon: '📄'},
```

Comment:
```suggestion
    {label: '대시보드', href: route.goAdminDashboard(), icon: '📊'},
    {label: '문서 관리', href: route.goAdminDocuments(), icon: '📄'},
```

이런 식으로 route 함수 추가해서 분리하는건 어떨까요? 통일성을 줄 수 있을것 같습니다.!

Reply:
감사합니다~ 여기 고치는 걸 깜빡했네요